apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -153,16 +147,10 @@ function WelcomeTour( { siteIntent }: { siteIntent?: string } ) {
 			tourRating: {
 				enabled: true,
 				useTourRating: () => {
```

Comment:
this is a very weird use of hooks

Reply:
They told us React was "just JavaScript", so why not hooks as config 😄

Reply:
haha, fair enough! It stood out since I hadn't seen hooks used like this before :P